README.md

```diff
@@ -105,7 +153,11 @@ d4p.daalfini()
 
 daal4py patching affects performance of specific Scikit-learn functionality listed below. In cases when unsupported parameters are used, daal4py fallbacks into stock Scikit-learn. These limitations described below. If the patching does not cover your scenarios, [submit an issue on GitHub](https://github.com/IntelPython/daal4py/issues).
 
-Scenarios that are already available in 2020.3 release:
+⚠️ We support optimizations for the last four versions of scikit-learn. The latest release of daal4py-2021.1 supports scikit-learn 0.21.X,
+0.22.X, 0.23.X and 0.24.X.
+
+<details><summary>[Click to expand] 🔥 Applying the daal4py patch will impact the following existing scikit-learn algorithms </summary>
```

Comment:
```suggestion
<details><summary>[Click to expand] 🔥 Applying the daal4py patch will impact the following existing scikit-learn algorithms: </summary>
```

I'd move "Click to expand" after that sentence, something like this:

---
🔥 Applying the daal4py patch will impact the following existing scikit-learn algorithms:

<details><summary>[Click to expand]</summary>

|Task|Functionality|Parameters support|Data support|
|:---|:------------|:-----------------|:-----------|
|Classification|**SVC**|All parameters except `kernel` = 'poly' and 'sigmoid'. | No limitations.|

</details>

---
also not sure about fire emoji for this :)

Reply:
Why?)

Reply:
emoji -- this is up to you, I just don't see how fire connects with the context/content

the other suggestion is just about making the hidden section a bit more visible to the readers